crowbar_framework/app/assets/stylesheets/bootstrap/_jumbotron.scss

```diff
@@ -4,37 +4,49 @@
 
 
 .jumbotron {
-  padding: $jumbotron-padding;
+  padding-top:    $jumbotron-padding;
```

Comment:
Properties should be ordered background-color, color, margin-bottom, padding-bottom, padding-top<br>Colon after property should be followed by one space